articles/151_node_to_participant_mapping.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: default
+title: Node to Participant mapping
+permalink: articles/node_to_participant_mapping.html
+abstract: This article analyzes the performance implications of enforcings a one-to-one mapping between ROS nodes and DDS participants, and propose alternative approaches.
```

Comment:
Can you clarify something for me? Are we only talking about composition? Or are we talking about trying to use the same participant regardless of how the node is written?

Reply:
The idea is that if you create a bunch of nodes or compose some nodes in the same process (by using manual composition or dynamically loading them in the same container), they should use the same participant by default.
A mechanism for using different participants in the same process should be available too (if we use one participant per context, creating a new context will be the way).